packages/yoroi-extension/.eslintignore

```diff
@@ -1,3 +1,11 @@
-node_modules
-/chrome/content-scripts
+ampli
+build
+chrome/content-scripts
+config
+dev
+docs
+flow
 flow-typed
+ledger
+node_modules
+translations
```

Comment:
Some of the directories added are not in the parameters of the eslint command in package.json at all, so why bother to add them here?

Reply:
That was an overkill when I locally tried to solve some issue with using `eslint .` , but it didn't work after all and I rolled it back, but left the ignores. Doesn't break anything so decided to leave it